src/js/configuration_panel/panel-events.js

```diff
@@ -113,7 +120,7 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             currentInputMenuItems[3].removeAttribute("disabled");
 
             // Reduce the width of the action dropdown.
-            dropdownMenu.style.width = "94%";
+            dropdownMenu.style.width = "96.5%";
```

Comment:
Why do you have to do this here and not in the stylesheet?

Reply:
It was added here so that when the user switches to an action that shows up the "Speed Factor" input box, the width of the action dropdown is reduced accordingly.

Reply:
I see.  Could you make a toggle class called "expanded" and do everything else in the stylesheet?  That seems preferable to hard-specifying the width deep in code.

Reply:
@the-t-in-rtf that's a good idea. I'll make the changes.